diff --git a/src/share/ma/keys_for_user b/src/share/ma/keys_for_user
index f48d5d3..b0c3cf5 100644
--- a/src/share/ma/keys_for_user
+++ b/src/share/ma/keys_for_user
@@ -44,7 +44,7 @@ export GNUPGHOME
 # extract user IDs from authorized_user_ids file
 IFS=$'\n'
 for line in $(meat "$authorizedUserIDs") ; do
-    su_monkeysphere_user ". ${SYSSHAREDIR}/common; keys_for_userid '$line'"
+    printf '%s\n' "$line" | su_monkeysphere_user ". ${SYSSHAREDIR}/common; keys_for_userid \"\$X\"" || true
 done
 
 }
